params/Augustinus vourinus.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 4
+    
+# e.g. "anas_krystallinus"    
+species.name <- ""anas_krystallinus" "
+    
+# e.g. "red"    
+color <- ""#FFFFFF" "
```

Comment:
I think the double quotes here will cause an error (also #ffffff is white, so it might not show up well).
```suggestion
color <- "#FFFFFF"
```